scripts/aws-config.json

```diff
@@ -0,0 +1,77 @@
+{
+  "IndexDocument": {
+    "Suffix": "index.html"
+  },
+  "ErrorDocument": {
+    "Key": "reference/404.html"
+  },
+  "RoutingRules": [
+    {
+      "Condition": {
+        "KeyPrefixEquals": "reference/installation"
+      },
+      "Redirect": {
+        "HostName": "crystal-lang.org",
+        "HttpRedirectCode": "301",
+        "Protocol": "https",
+        "ReplaceKeyPrefixWith": "install"
+      }
+    },
+    {
+      "Condition": {
+        "KeyPrefixEquals": "reference/overview/hello_world.html"
```

Comment:
Hm why is this redirect of a particular page here? Isn't this possible to achieve through redirects plugin?

Reply:
Yeah, we should probably do that. But that's what we have currently. Well, it actually didn't work until yesterday because the key prefixes started with an `/` which doesn't match 🤷 
So I suppose we might as well drop it.

Btw. is it even possible to have redirects from non-versioned paths?

Reply:
> is it even possible to have redirects from non-versioned paths?

I think that will happen through first redirecting to a particular version and then that versioned path handling its redirect.